package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "lint": "tslint src/**/*.ts",
     "test": "karma start",
-    "build": "echo 'Building...' && ./node_modules/.bin/ngc -p tsconfig.json"
+    "build": "echo 'Building...' && ./node_modules/.bin/ngc -p tsconfig.json",
+    "update-demo": "cd demo && ng build --prod --bh /angular2gridster/ && ./../node_modules/.bin/gh-pages -d ./dist"
```

Comment:
`cd demo && ng build...` command required `angular-cli` installed as global package. It's not best option.

`./../node_modules/.bin/gh-pages -d ./dist` => `(npm bin)/gh-pages -d ./dist` ?

Reply:
You mean `$(npm bin)` - you are right, it's better. Since this command is strictly related with demo app, I moved this command to demo/package.json.